Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/Validation/PI.mo

```diff
@@ -0,0 +1,318 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.Validation;
+model PI "Test model for PI"
+  Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.PI PI
+    "Calculate the parameters for a PI controller"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.TimeTable RefeDat(table=[0,1,
```

Comment:
it's hard to maintain the large table in .mo file. Can you please see the model `Buildings.Experimental.DHC.Loads.BaseClasses.Examples.BaseClasses.BuildingTimeSeries` the instance of 'loa` which also shows the time series data, but saved the file in .mos file?

Reply:
Thanks for the information but are we allowed to use blocks outside of the OBC folder?

Reply:
In the validation package, it is ok to use the outside blocks.

Reply:
The validation packages should also only use CDL; you can move the block outside of the OBC package.
(This would be the first time we don't use CDL inside Buildings.OBC.)
@JayHuLBL : The test
https://github.com/lbl-srg/modelica-buildings/blob/b3bc444b85a131748b3a8f9f9b469e9bb0116ae9/Buildings/Resources/Scripts/travis/Makefile#L26
should fail if non-CDL is used in an validation or an example.

Reply:
Thanks for the explanation